### PR TITLE
[CLI-2014] Add Billing v1 sdk

### PR DESCRIPTION
**Pull request by MuweiHe**
Put every go file and the structs files in root as one package. Removed cc-structs dependencies.

### CODE REVIEW

File: interfaces.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package ccloud
+
+import (
+	"context"
+)
+
+// Billing service allows getting billing information for an org in Confluent Cloud
+type Billing interface {
```
Comment (brianstrauch) [inline interfaces.go:8]: Are we including this `interface` to generate mocks?

File: version.go [hunk 2]
```diff
@@ -0,0 +1,3 @@
+package ccloud
+
+const SDKVersion = "v0.1.0"
```
Comment (brianstrauch) [inline version.go:3]: We need to make sure our CI bumps this version automatically. This should be updated manually in the meantime.

**Review by brianstrauch — CHANGES_REQUESTED** [review]

File: interfaces.go (hunk 1 above)
Comment (MuweiHe) [reply interfaces.go:8]: yes this is needed

File: billing_structs.go [hunk 3]
```diff
@@ -0,0 +1,3055 @@
...
+	}
+	l = len(m.StripeObjectId)
+	if l > 0 {
+		n += 1 + l + sovBilling(uint64(l))
+	}
+	if m.XXX_unrecognized != nil {
+		n += len(m.XXX_unrecognized)
+	}
+	return n
+}
```
[the remark this answers is not in the document]
Comment (MuweiHe) [reply billing_structs.go:276]: yes it's being called by other marshal/unmarshal functions

File: billing_structs.go [hunk 4]
```diff
@@ -0,0 +1,3055 @@
...
+	XXX_unrecognized     []byte           `json:"-"`
+	XXX_sizecache        int32            `json:"-"`
+}
+
+func (m *PromoCodeClaim) Reset()         { *m = PromoCodeClaim{} }
+func (m *PromoCodeClaim) String() string { return proto.CompactTextString(m) }
+func (*PromoCodeClaim) ProtoMessage()    {}
+func (*PromoCodeClaim) Descriptor() ([]byte, []int) {
+	return fileDescriptor_f6f5cf017d9954b9, []int{48}
+}
```
[the remark this answers is not in the document]
Comment (MuweiHe) [reply billing_structs.go:59]: Maybe not Reset but we need `String` and `ProtoMessage`

File: billing_structs.go [hunk 5]
```diff
@@ -0,0 +1,3055 @@
...
+
+type TaxId struct {
+	// details of taxId definition is here:
+	// https://stripe.com/docs/api/customer_tax_ids/object
+	TypeEnum             string   `protobuf:"bytes,1,opt,name=type_enum,json=typeEnum,proto3" json:"type_enum,omitempty" db:"type_enum,omitempty" url:"type_enum,omitempty"`
+	TaxId                string   `protobuf:"bytes,2,opt,name=tax_id,json=taxId,proto3" json:"tax_id,omitempty" db:"tax_id,omitempty" url:"tax_id,omitempty"`
+	StripeObjectId       string   `protobuf:"bytes,3,opt,name=stripe_object_id,json=stripeObjectId,proto3" json:"stripe_object_id,omitempty" redact:"-" db:"stripe_object_id,omitempty" url:"stripe_object_id,omitempty"`
+	XXX_NoUnkeyedLiteral struct{} `json:"-"`
+	XXX_unrecognized     []byte   `json:"-"`
+	XXX_sizecache        int32    `json:"-"`
```
[the remark this answers is not in the document]
Comment (MuweiHe) [reply billing_structs.go:38]: if i remember correctly those `xxx` stuff are needed if we want to call e.g. `json.Marshal` when importing them into another repo. This is fine to delete for billing I think, but for other sdk we need to check if no `json.---` methods are called on them in cli repo.

File: billing_structs.go [hunk 6]
```diff
@@ -0,0 +1,3055 @@
...
+
+	_ "github.com/confluentinc/proto-go-setter"
+	_ "github.com/envoyproxy/protoc-gen-validate/validate"
+	_ "github.com/gogo/protobuf/gogoproto"
+	proto "github.com/gogo/protobuf/proto"
+	types "github.com/gogo/protobuf/types"
+)
+
+type TaxAddress struct {
+	Line1 string `protobuf:"bytes,1,opt,name=line1,proto3" json:"line1,omitempty" db:"line1,omitempty" url:"line1,omitempty"`
```
[the remark this answers is not in the document]
Comment (MuweiHe) [reply billing_structs.go:17]: I think it's ok to leave them here.. Deleting each tag manually can be slow. I want to do the migration first and if needed I can come back to clean up that..

File: billing_structs.go (hunk 5 above)
[the remark this answers is not in the document]
Comment (brianstrauch) [reply billing_structs.go:38]: Huh, interesting. Guess we need them?

**Review by brianstrauch — APPROVED** [review]
Looks good, as long as my two remaining comments are addressed.

**Pull request merged**